R/conga.R

```diff
@@ -1,16 +1,18 @@
-#' Calculate continuous overall net glycemic action (CONGA)
+#' Continuous Overall Net Glycemic Action (CONGA)
 #'
 #' @description
 #' The function conga produces a CONGA values a tibble object.
-#' conga currently only supports calculation of CONGA24.
+#' conga now supports calculation of any n hours apart.
 #'
 #' @usage
-#' conga(data, tz = "")
+#' conga(data, tz = "", n = 1)
 #'
 #' @param data DataFrame object with column names "id", "time", and "gl".
 #'
 #' @param tz A character string specifying the time zone to be used. System-specific (see \code{\link{as.POSIXct}}), but " " is the current time zone, and "GMT" is UTC (Universal Time, Coordinated). Invalid values are most commonly treated as UTC, on some platforms with a warning.
 #'
+#' @param n An integer specifying how many hours prior to an observation should be used in the CONGA calculation
```

Comment:
Add what is the default value. Do we want the default to be 24 for consistency with old code?